media-libs/harfbuzz/harfbuzz-9.0.0.recipe

```diff
@@ -79,6 +80,24 @@ REQUIRES_devel="
 	devel:libicuuc$secondaryArchSuffix
 	"
 
+PROVIDES_tools="
+	harfbuzz${secondaryArchSuffix}_tools = $portVersion compat >= 0.9
```

Comment:
is this a typo? (should that be 9.0 and not 0.9)

Reply:
That was me copying the other provides lines

Reply:
Sorry, I didn't notice the unchanged part above. Maybe factor it out in a separate `portVersionCompat` variable to not repeat this twice.

Not all recipes do this, but I think it would be clearer that way.

Reply:
OK, done